src/olympia/users/tests/test_models.py

```diff
@@ -8,10 +8,13 @@
 from django.test.client import RequestFactory
 from django.test.utils import override_settings
 from ipaddress import IPv4Address
+import os.path
+from olympia.amo.tests.test_helpers import get_uploaded_file
```

Comment:
I guess all the imports for this file are ignored by ruff because of the `# noqa`s but can we have these in roughly the correct place 😉

Reply:
Oh wow that's weird. I'm used to `ruff` autofixing the imports so now I just stuff them wherever and hit save to have them correctly set. I'll fix this one manually but we'll want to look at why we have these `# noqa` and if we can avoid them, having `ruff` ignore the whole import block is bad...

Reply:
c86eb51faaffc90839a78851e0ea0eeef9922089